docs/pages/reference/cli/tctl.mdx

```diff
@@ -1007,6 +1007,45 @@ $ tctl request approve [token]
 $ tctl request approve request-id-1, request-id-2
 ```
 
+## tctl request create
+
+Create a pending Access Request.
+
+```code
+$ tctl request create <username>
+```
+
+### Arguments
+
+- `<username>` - Name of target user (required).
+
+### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+|`roles`|none|Comma-separated list of strings|Roles to be requested|
+|`resource`|none|Comma-separated list of strings|Resource IDs to be requested|
+|`reason`|none|String|Optional reason message|
+|`dry-run`|none|Boolean|Don't actually generate the Access Request|
+
+Use the `dry-run` flag if you want to validate whether Teleport can create an
+Access Request for the user in the `username` argument, given the user's static
+roles.
+
+### Global flags
+
+These flags are available for all commands `--debug, --config`. Run
+`tctl help <subcommand>` or see the [Global Flags section](#tctl-global-flags).
+
+### Examples
+
+Create an Access Request for user `myuser` for the `prod` role, providing a
```

Comment:
Is this only available with RoleAdmin or does it work with remote credentials too?

Reply:
If it works I don't think there's any need to mention it.

Thanks for double checking. I recall some funny things with the `tctl` flows due to the way we originally implemented access requests in community edition.